src/OpenLoco/Objects/VehicleObject.h

```diff
@@ -39,52 +39,52 @@ namespace OpenLoco
     }
 
 #pragma pack(push, 1)
-    struct vehicle_object_sound_1
+    struct VehicleObjectSoundFriction
     {
-        uint8_t sound_object_id;
-        uint32_t var_01;
-        uint8_t var_05;
-        uint16_t var_06;
-        uint8_t var_08;
-        uint8_t var_09;
-        uint8_t var_0A;
+        uint8_t soundObjectId;     // 0x0
+        Speed32 minSpeed;          // 0x1 below this speed no sound created
+        uint8_t speedFreqFactor;   // 0x5
+        uint16_t baseFrequency;    // 0x6
+        uint8_t speedVolumeFactor; // 0x8
+        uint8_t baseVolume;        // 0x9
+        uint8_t maxVolume;         // 0xA
     };
 
-    struct vehicle_object_sound_2
+    struct VehicleObjectSoundEngine1
     {
-        uint8_t sound_object_id;
-        uint16_t var_01;
-        uint8_t var_03;
+        uint8_t soundObjectId;     // 0x0
+        uint16_t defaultFrequency; // 0x1
+        uint8_t defaultVolume;     // 0x3
         uint16_t var_04;
         uint8_t var_06;
         uint16_t var_07;
         uint8_t var_09;
-        uint16_t var_0A;
-        uint16_t var_0C;
-        uint8_t var_0E;
-        uint8_t var_0F;
-        uint8_t var_10;
+        uint16_t freqIncreaseStep;  // 0xA
+        uint16_t freqDecreaseStep;  // 0xC
+        uint8_t volumeIncreaseStep; // 0xE
+        uint8_t volumeDecreaseStep; // 0xF
+        uint8_t speedFreqFactor;    // 0x10
     };
 
-    struct vehicle_object_sound_3
+    struct VehicleObjectSoundEngine2
```

Comment:
I get that you'd probably like to keep the `VehicleObjectSound` part as a base, but now it reads like `Sound Engine` instead. I suggest:
```suggestion
    struct VehicleObjectEngineSound2
```